modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -444,16 +498,42 @@ void ONNXImporter::populateNet(Net dstNet)
             }
             else if (is_const_0 || is_const_1)
             {
-                Mat blob = getBlob(node_proto, constBlobs, is_const_0 ? 0 : 1);
-                blob = blob.reshape(1, 1);
+                int const_id = is_const_0 ? 0 : 1;
+                Mat blob = getBlob(node_proto, constBlobs, const_id);
                 if (blob.total() == 1) {
                     layerParams.type = "Power";
                     layerParams.set("shift", (isSub ? -1 : 1) * blob.at<float>(0));
                 }
                 else {
-                    layerParams.type = "Scale";
-                    layerParams.set("bias_term", true);
-                    layerParams.blobs.push_back((isSub ? -1 : 1) * blob);
+                    int blobTotal = blob.total();
+                    MatShape inpShape = outShapes[node_proto.input(1 - const_id)];
+                    if (blobTotal > 1 && blobTotal == total(inpShape))
```

Comment:
Why we need to check `blobTotal > 1`?

Reply:
Because eltwise layer can work with `inputs[0].size() >= 2 `